entries/entry03.md

```diff
@@ -1,8 +1,48 @@
 # Entry 3
-##### X/X/XX
+##### 1/5/20
 
-Text
+#### Progress
+Since the second blog entry, I mainly focused on studying [Arduino](https://www.arduino.cc/) code.
+During my exploration of Arduino's code, I learned about the code for RFID RC522, such as creating an MFRC522 instance:
```

Comment:
But where specifically did you learn about RC522? **That** is where you should link to.